internal/testrunner/runners/system/servicedeployer/kubernetes.go

```diff
@@ -144,14 +149,56 @@ func findKubernetesDefinitions(definitionsDir string) ([]string, error) {
 func installElasticAgentInCluster() error {
 	logger.Debug("install Elastic Agent in the Kubernetes cluster")
 
-	locationManager, err := locations.NewLocationManager()
+	elasticAgentManagedYaml, err := getElasticAgentYaml()
 	if err != nil {
-		return errors.Wrap(err, "can't locate Kubernetes file for Elastic Agent in ")
+		return errors.Wrap(err, "can't retrieve Kubernetes file for Elastic Agent")
 	}
 
-	err = kubectl.Apply(locationManager.KubernetesDeployerAgentYml())
+	err = kubectl.ApplyStdin(elasticAgentManagedYaml)
 	if err != nil {
 		return errors.Wrap(err, "can't install Elastic-Agent in Kubernetes cluster")
 	}
 	return nil
 }
+
+// downloadElasticAgentManagedYAML will download a url from a path and return the response body as a string.
+func downloadElasticAgentManagedYAML(url string) (string, error) {
+	// Get the data
+	resp, err := http.Get(url)
+	if err != nil {
+		return "", errors.Wrapf(err, "failed to get file from url %s", url)
+	}
+	defer resp.Body.Close()
+
+	// Convert to string
+	b, err := io.ReadAll(resp.Body)
```

Comment:
is it necessary to go with string?

Reply:
It is not necessary. I did that in order to use it later in Regexp `ReplaceAllString`. 
But I can instead use `ReplaceAll` that takes a slice of bytes as input.

Reply:
> But I can instead use ReplaceAll that takes a slice of bytes as input.

Sounds good to me